Sourcery/Templates/Diffable.stencil

```diff
@@ -2,12 +2,13 @@
 extension {{ type.name }}{% if not type.supertype.implements.AutoDiffable %}: Diffable{% endif %} {
     {% if type.based.Type %}override {% endif %}func diffAgainst(_ object: Any?) -> DiffableResult {
         let results = DiffableResult()
-        {% for variable in type.storedVariables %}{% if not variable.annotations.skipEquality %}{% if forloop.first %}guard let rhs = object as? {{ type.name }} else {
+        guard object is {{ type.name }} else {
```

Comment:
@krzysztofzablocki I remember you saying that it will be still good to check for type, even if there are no variables to compare. Without filters in `if` tag I could not come up with a better approach to avoid warnings for unused `rhs` (what was the reason for previous change in this template).